creatorkit/Toolbox.md

```diff
@@ -0,0 +1,135 @@
+![Creator Logo](../creatorlogo.png)
+
+# Toolbox
+
+The toolbox is a reference page for you to use to set up your kit. Follow the steps listed in the project pages and refer back to this page for more detail on individual steps.
+
+## Hardware
+
+* Creator Kit
+* Ubuntu 14.04 64-bit PC/Laptop
+
+## Powering Ci40 + Clicker boards
+
+There are 2 ways to power your Ci40 board.
+
+* Using a power supply
+* Using a microUSB cable connected to the port next to the power supply port
+
+There are also 2 ways to power Clicker boards
+
+* 2 AAA Batteries
+* MicroUSB
+
+## Getting the code
+
+You can get pre-built releases or source code of the Creator Kit projects to work with. The Releases will allow you to set up a working system quickly without having to build any code. The source code will allow you to inspect/extend the projects and build the releases yourself.
+
+[Releases](http://github.com/CreatorKit)
+
+[Source Code](http://github.com/CreatorKit)
+
+### Connecting to Ci40 via Serial
+
+![serial](serialport.resized.jpg)
+
+The next step is to connect to your Ci40 using a serial terminal. Connect a microUSB-USB A cable to the port within the Mikrobus connector and into your PC. You can use any serial terminal program you like with the baud rate set to 115200. The following is an example using miniterm in Ubuntu
+
+<code>$ sudo miniterm.py /dev/ttyUSB0 -b 115200</code>
+
+To determine the address to use (in this case /dev/ttyUSB0) you can run
+
+<code>$ ls /dev/ttyU*</code>
+
+Once connected, you should have access to a terminal.
+
+## Creating a Bootable USB Drive/microSD Card
+
+Note:
+
+* You cannot boot from USB Drive while also powering the board via microUSB as they both use the same connector (the one next to the power socket). You will need to power the board using a power supply instead, or use a microSD card as your boot device.
+* To connect a USB pen drive, you need a USB A to microUSB OTG cable.
```

Comment:
If this microUSB correct, then change the above one MicroUSB. but point is lets be consistent across the documentation.